src/Build/Globbing/Extensions/IMSBuildGlobExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//-----------------------------------------------------------------------
+
+
+using System.Collections.Generic;
+
+namespace Microsoft.Build.Globbing.Extensions
+{
+    /// <summary>
+    /// Extensions for <see cref="IMsBuildGlob"/>
+    /// </summary>
+    public static class IMsBuildGlobExtensions
```

Comment:
Not an interface so remove the I from the name/file. Also for more consistency, use `MSBuildGlobExtensions` (uppercase `S`) even though resharper doesn't like it.